website/docs/advanced/strategy-constraints.md

```diff
@@ -165,6 +165,12 @@ If the strategy constraint uses a [**standard Unleash Context field**](../user_g
 
 If the strategy constraint uses a [**custom context field**](../user_guide/unleash_context#custom-context-fields), use the Unleash Context's `properties` field. Use the name of the custom context field as a key and set the value to your desired string.
 
+If you set a context field to a value that the SDKs cannot parse correctly for a chosen constraint operator, the strategy constraint will evaluate to false.
+In other words: if you have a strategy constraint operator that expects a number, such as `NUM_GT`, but you set the corresponding context field to a string value, then the expression will be false: `"some string"` is **not** greater than `5`.
+This value can still be negated as explained in [the section on negating values](#constraint-negation).
```

Comment:
@sighphyre Did you want something like this about the invalid values?

Reply:
This is great, thank you!